input/kafkamdm/kafkamdm.go

```diff
@@ -221,6 +236,14 @@ func (k *KafkaMdm) consumePartition(topic string, partition int32, partitionOffs
 			if err := offsetMgr.Commit(topic, partition, currentOffset); err != nil {
 				log.Error(3, "kafka-mdm failed to commit offset for %s:%d, %s", topic, partition, err)
 			}
+			partitionOffset[partition].Set(int(currentOffset))
+			offset, err := k.client.GetOffset(topic, partition, -1)
```

Comment:
instead of `-1` wouldn't it be more readable to put `sarama.OffsetNewest` there?